### PR TITLE
Add note about memory requirements in CAP.

### LINKED ISSUE #903: Warning about schedulers and stacks requiring different memory settings.
Per https://github.com/cloudfoundry-incubator/kubecf/issues/914, it has been confirmed that deploying the same app across CAP will require different memory requirements.

This means the same app on schedulers like Diego or Eirini, or stacks like cflinuxfs3 or sle15, would need tests to ensure enough memory is allocated. For example, deploying a Ruby app will require ~50% more memory when deploying on sle15 versus cflinuxfs3. We didn't test every permutation or every language but this gives a sign that there isn't an absolute memory setting for your app.

### PR DESCRIPTION
**Pull request by KucharczykL**
Closes #903 

I have added the note into the Eirini section and in the Deployment Notes. Let me know if there are others or if these two would be better suited somewhere else.

### CODE REVIEW

File: xml/cap_depl_admin_notes.xml [hunk 1]
```diff
@@ -26,6 +26,11 @@
  </sect1>
  <sect1>
   <title>Important Changes</title>
+  <para>
+   Different schedulers, such as like Diego and Eirini, and different stacks, such
```
Comment (btat) [inline xml/cap_depl_admin_notes.xml:30]: Only one of `such as` or `like` needed

File: xml/cap_depl_admin_notes.xml [hunk 2]
```diff
@@ -26,6 +26,11 @@
...
+   as cflinuxfs3 or sle15, have different memory requirements. Not every combination
```
Comment (btat) [inline xml/cap_depl_admin_notes.xml:31]: The stacks should be enclosed with literal tags.

File: xml/cap_depl_eirini.xml [hunk 3]
```diff
@@ -23,6 +23,11 @@
   and <link xlink:href="http://eirini.cf"/>
  </para>
 
+ <para>
+  Different schedulers and stacks have different memory requirements.
+  Not every combination is tested so there is no universal memory setting for &cap;.
```
Comment (btat) [inline xml/cap_depl_eirini.xml:28]: Applies to both files:

Need to mention this is the memory of applications. (Set via `-m` during `cf push` or in the app's manifest https://docs.cloudfoundry.org/devguide/deploy-apps/manifest-attributes.html#memory)

The `Not every combination is tested so there is no universal memory setting for &cap;.` sentence should probably be changed to indicate the memory difference can vary depending on the app deployed and it's up to the user to adjust the setting based on their app.

File: xml/cap_depl_admin_notes.xml (hunk 1 above)
Comment (KucharczykL) [reply xml/cap_depl_admin_notes.xml:30]: Thanks for noticing my silly typo!:D

**Pull request merged**